ReadASCIIApp/Db/get_metadata.db

```diff
@@ -44,6 +45,6 @@ record(aSub, "$(OUT):_UNITS")
     field(INPE, "$(OUT):$(OUTF=VAL):_META:DEF CP")
     field(FTE, "STRING")
       
-    field(OUTA, "$(OUT).$(OUTF=VAL)")
+    field(OUTA, "$(OUT).$(OUTF=VAL) $(OUTPP=NP)")
```

Comment:
Default should be `NPP` rather than `NP` (although I think this will work because `NP` is not valid so EPICS will ignore it and default to `NPP` anyway).